Bryn Mawr Classical Review.js

```diff
@@ -134,23 +154,13 @@ var testCases = [
 				"itemType": "journalArticle",
 				"creators": [
 					{
-						"firstName": "Christina S.",
-						"lastName": "Kraus",
-						"creatorType": "author"
-					},
-					{
-						"firstName": "Anthony",
-						"lastName": "Grafton",
-						"creatorType": "reviewedAuthor"
-					},
-					{
-						"firstName": "Glenn W.",
-						"lastName": "Most",
-						"creatorType": "reviewedAuthor"
+ 						"firstName": "Keely Elizabeth",
+ 						"lastName": "Heuer",
+ 						"creatorType": "author"
 					},
 					{
-						"firstName": "Salvatore",
-						"lastName": "Settis",
+						"firstName": "Rolf",
+						"lastName": "Hurschmann",
```

Comment:
Yes, the second test case is 04, not 44. It looks like the original test case had both 04 and 44.

The other test case is http://bmcr.brynmawr.edu/2010/2010-01-02.html, again inherited from the original.

To confuse things, Kraus was the author of both the current first example and the mistaken second example. (I've not dug further back into the history of the file to see what happened.)